static/src/javascripts/projects/facia/modules/onwards/search-tool.js

```diff
@@ -0,0 +1,259 @@
+// @flow
+import bean from 'bean';
+import $ from 'lib/$';
+import fetchJson from 'lib/fetch-json';
+import mediator from 'lib/mediator';
+import reportError from 'lib/report-error';
+import bonzo from 'bonzo';
+
+interface WeatherSearchOptions {
+    container: bonzo,
+    apiUrl: string,
+}
```

Comment:
Mainly because I didn't know Flow well enough to realise that was an alternative. A `type` seems more idiomatic in this case, so I'll use that.